clients/commons/utils/validationUtils.ts

```diff
@@ -0,0 +1,56 @@
+const PASSWORD_MINIMUM_LENGTH = Number(process.env.PASSWORD_MINIMUM_LENGTH)
+
+type ValidationResult = {
+  message: string;
+  isValid: boolean;
+}
+
+export const validatePassword = (
+  password: string,
+  confirmPassword: string
+): {
+  passwordResults: ValidationResult[];
+  confirmPasswordResult: ValidationResult;
+} => {
+  const passwordResults: ValidationResult[] = []
```

Comment:
i suggested an implementation on ebrahim's pr, please have a look

Reply:
Updated in 822b3b468a429c8071ce249b63e8ab0ca5ed0cc5